README.md

```diff
@@ -64,30 +58,28 @@ a package developer want/need to run this command?
 
 _Context: package_
 
-### `elastic-package lint`
 
-Use this command to validate the contents of a package.
+### `elastic-package check`
+
+Use this command to run the `format`, `lint`, and `build` commands all at once, in that order.
```

Comment:
... and the future the "test" subcommand.

I wonder if we should include "test" in the "check" chain as it requires the cluster to be up and running.

Reply:
Let's keep the README about current state as much as possible. We can always update it easily as/when we make enhancements or changes.

I could go either way on including `test` in `check` but I slightly favor keeping it out. This way `check` can run relatively fast and can be run as an earlier stage in CI.